stamina-testkit/src/main/scala/stamina/testkit/StaminaTestKit.scala

```diff
@@ -59,12 +62,9 @@ trait StaminaTestKit { self: org.scalatest.WordSpecLike ⇒
       }
     }
 
-    implicit def eitherToTry[B](either: Either[base64.Decode.Failure, B]): Try[B] = {
```

Comment:
This implicit conversion wasn't private, so it's up to you if you consider it breaking.